### PR TITLE
Update command-reference.md

### PR DESCRIPTION
**Pull request by Celia18305**
To fix majority of issue #299

### CODE REVIEW

**Comment by Celia18305** [comment]
@chenzhitong please kindly review and supplement information for new added commands if need be.

File: docs/command-reference.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!-- markdownlint-enable -->
 # Neo-Express N3 Command Reference
 
-> Note, this is the command reference for Neo-Express 3.0, targeting N3.
+> Note: This is the command reference for Neo-Express 3.6, targeting N3.
 > The [Command Reference](legacy-command-reference.md) for the Neo Legacy
```
Comment (chenzhitong) [inline docs/command-reference.md:5]: The link is 404.

**Comment by chenzhitong** [comment]
[invocation file](https://github.com/ngdenterprise/design-notes/blob/master/NDX-DN12%20-%20Neo%20Express%20Invoke%20Files.md) is 404.
[string arguments in an invocation file](https://github.com/ngdenterprise/design-notes/blob/master/NDX-DN12%20-%20Neo%20Express%20Invoke%20Files.md#args-property). is 404.

File: docs/command-reference.md [hunk 2]
```diff
@@ -486,17 +520,217 @@ Arguments:
...
+Arguments:
+  Contract            NFT Contract (Symbol or Script Hash)
+  Account             Account to show NFT (Format: Script Hash, Address, Wallet name)
+
+Options:
+  -i|--input <INPUT>  Path to neo-express data file
+```
+
+The `show nft` displays the content of an NFT contract for a specified account.
+
```
Comment (chenzhitong) [inline docs/command-reference.md:618]: It is suggested to supplement:
The output will consist of TokenId in Base64 format and TokenId in big-endian Hex string format.

File: docs/command-reference.md [hunk 3]
```diff
@@ -486,17 +520,217 @@ Arguments:
...
+### neoxp candidate list
+
+```
+Usage: neoxp candidate list [options]
+
+Options:
+  -t|--trace          Enable contract execution tracing
+  -j|--json           Output as JSON
+  -i|--input <INPUT>  Path to neo-express data file
+```
```
Comment (chenzhitong) [inline docs/command-reference.md:662]: The results include the public key of the candidate and the number of ballots cast.

File: docs/command-reference.md [hunk 4]
```diff
@@ -616,6 +850,35 @@ The commands supported in a batch file include:
...
+## neoxp execute
+
+```
+Usage: neoxp execute [options] <InputText>
+
+Arguments:
+  InputText                           A neo-vm script (Format: HEX,                                      BASE64, Filename)
+
+Options:
+  -a|--account <ACCOUNT>              Account to pay invocation GAS                                      fee
```
Comment (chenzhitong) [inline docs/command-reference.md:862]: format error

**Comment by chenzhitong** [comment]
Add `TransferNFT` in neoxp batch command list

**Comment by Celia18305** [comment]
Updated. Please double check.

**Review by chenzhitong — APPROVED** [review]

**Review by Liaojinghui — APPROVED** [review]

**Pull request merged**